src/App.tsx

```diff
@@ -61,6 +60,7 @@ export const App = () => {
         onUpdateConversationTitle={onUpdateConversationTitle}
         isSideBarOpen={isSideBarOpen}
         SideBarLogo={sideBarLogo}
+        styles={styles}
```

Comment:
Passing the app module styles (which control how sidebar effects chat interface width) into the lib component so that these children can be interactive.